Fix failed TestManifestDirectoryPath

```
2019-09-20T17:46:28.5246958Z --- FAIL: TestManifestDirectoryPath (0.00s)
2019-09-20T17:46:28.5247605Z     --- FAIL: TestManifestDirectoryPath/same_directory_level (0.00s)
2019-09-20T17:46:28.5248236Z         workspace_test.go:265: 
2019-09-20T17:46:28.5248894Z             	Error Trace:	workspace_test.go:265
2019-09-20T17:46:28.5249217Z             	Error:      	Not equal: 
2019-09-20T17:46:28.5249522Z             	            	expected: "test/ecs"
2019-09-20T17:46:28.5249820Z             	            	actual  : "test\\ecs"
2019-09-20T17:46:28.5250431Z             	            	
2019-09-20T17:46:28.5250872Z             	            	Diff:
2019-09-20T17:46:28.5251327Z             	            	--- Expected
2019-09-20T17:46:28.5251853Z             	            	+++ Actual
2019-09-20T17:46:28.5252197Z             	            	@@ -1 +1 @@
2019-09-20T17:46:28.5252483Z             	            	-test/ecs
2019-09-20T17:46:28.5252764Z             	            	+test\ecs
2019-09-20T17:46:28.5253088Z             	Test:       	TestManifestDirectoryPath/same_directory_level
2019-09-20T17:46:28.5253743Z     --- PASS: TestManifestDirectoryPath/same_directory (0.00s)
2019-09-20T17:46:28.5254131Z     --- FAIL: TestManifestDirectoryPath/several_levels_deep (0.00s)
2019-09-20T17:46:28.5254697Z         workspace_test.go:265: 
2019-09-20T17:46:28.5255214Z             	Error Trace:	workspace_test.go:265
2019-09-20T17:46:28.5255634Z             	Error:      	Not equal: 
2019-09-20T17:46:28.5255938Z             	            	expected: "test/ecs"
2019-09-20T17:46:28.5256236Z             	            	actual  : "test\\ecs"
2019-09-20T17:46:28.5256844Z             	            	
2019-09-20T17:46:28.5257131Z             	            	Diff:
2019-09-20T17:46:28.5257428Z             	            	--- Expected
2019-09-20T17:46:28.5257715Z             	            	+++ Actual
2019-09-20T17:46:28.5257996Z             	            	@@ -1 +1 @@
2019-09-20T17:46:28.5258299Z             	            	-test/ecs
2019-09-20T17:46:28.5258576Z             	            	+test\ecs
2019-09-20T17:46:28.5258876Z             	Test:       	TestManifestDirectoryPath/several_levels_deep
```

--- a/internal/pkg/workspace/workspace_test.go
+++ b/internal/pkg/workspace/workspace_test.go
@@ -192,7 +192,8 @@ func TestWriteManifest(t *testing.T) {
 }
 
 func TestManifestDirectoryPath(t *testing.T) {
-	var platformDependentExpectedManifestDir = filepath.FromSlash("test/ecs")
+	// turn "test/ecs" into a platform-dependent path
+	var manifestDir = filepath.FromSlash("test/ecs")
 
 	testCases := map[string]struct {
 		expectedManifestDir string
@@ -202,7 +203,7 @@ func TestManifestDirectoryPath(t *testing.T) {
 		mockFileSystem      func(appFS afero.Fs)
 	}{
 		"same directory level": {
-			expectedManifestDir: platformDependentExpectedManifestDir,
+			expectedManifestDir: manifestDir,
 			workingDir:          filepath.FromSlash("test/"),
 			mockFileSystem: func(appFS afero.Fs) {
 				appFS.MkdirAll("test/ecs", 0755)
@@ -210,7 +211,7 @@ func TestManifestDirectoryPath(t *testing.T) {
 		},
 
 		"same directory": {
-			expectedManifestDir: platformDependentExpectedManifestDir,
+			expectedManifestDir: manifestDir,
 			workingDir:          filepath.FromSlash("test/ecs"),
 			mockFileSystem: func(appFS afero.Fs) {
 				appFS.MkdirAll("test/ecs", 0755)
@@ -218,7 +219,7 @@ func TestManifestDirectoryPath(t *testing.T) {
 		},
 
 		"several levels deep": {
-			expectedManifestDir: platformDependentExpectedManifestDir,
+			expectedManifestDir: manifestDir,
 			workingDir:          filepath.FromSlash("test/1/2/3/4"),
 			mockFileSystem: func(appFS afero.Fs) {
 				appFS.MkdirAll("test/ecs", 0755)
@@ -244,7 +245,7 @@ func TestManifestDirectoryPath(t *testing.T) {
 		},
 
 		"uses precomputed manifest path": {
-			expectedManifestDir: platformDependentExpectedManifestDir,
+			expectedManifestDir: manifestDir,
 			workingDir:          filepath.FromSlash("/"),
 			mockFileSystem:      func(appFS afero.Fs) {},
 			presetManifestDir:   filepath.FromSlash("test/ecs"),

